assembl/alembic/versions/4f529b59026b_add_semantic_analysis_to_section_type.py

```diff
@@ -0,0 +1,29 @@
+"""Add semantic analysis to section type
+
+Revision ID: 4f529b59026b
+Revises: d813ab110e00
```

Comment:
please update the comment to 4227dfe5456c

Reply:
done in https://github.com/assembl/assembl/pull/643/commits/46fdad6d00e26a8731dacb9b0348556b476fdf71

Reply:
@vincentfretin can you confirm that we have nothing to do in our side ? Thx

Reply:
Yes, it's done.